pyproject.toml

```diff
@@ -125,3 +125,76 @@ wheel.exclude = [
     # No need to package template files
     "**.py.in"
 ]
+
+[tool.ruff.per-file-target-version]
+# By default, ruff uses project.requires-python to infer the project-wide
+# python version (certain linting/formatting options may be enabled or disabled
+# based on this choice). Here, we are overwriting this version for our
+# developer/build scripts
+"scripts/**" = "py37"
+"tests/scripts/**" = "py37"
+
+[tool.ruff.lint]
+# By default, Ruff enables Flake8's F rules, along with a subset of the E rules,
+# omitting any stylistic rules that overlap with the use of a formatter, like
+# ruff format
+select = ["E4", "E7", "E9", "F"]
+# our ignore-list is based on our old flake8 ignore-list
+ignore = ["E402", "E502", "E701", "E731"]
+
+exclude = [
+    # these are based on flake8's exclude-list
+    "src/python/pygrackle/api.py",
```

Comment:
This is gone, right?
```suggestion
```

Reply:
We need to merge the most recent changes from newchem-cpp, first. If you want to do that, I would be happy to make this change.

Reply:
Sure thing, done. There was a supposed merger conflict with the README file. git claimed that `README.md` had been modified in `newchem-cpp` after being deleted in `main`. I couldn't find any evidence of this so my merger resolution was just to delete it again and not make any changes to `README.rst`.